apps/SageAccountWeb/src/components/ResetPassword.tsx

```diff
@@ -129,44 +151,76 @@ export const ResetPassword = (props: ResetPasswordProps) => {
                   type="submit"
                   fullWidth
                   onSubmit={handleChangePasswordWithToken}
+                  sx={buttonSx}
+                  disabled={!newPassword || !confirmPassword}
                 >
                   Change Password
                 </Button>
               </form>
             </>
           ) : (
-            <>
-              <Typography variant="headline2">Reset your Password</Typography>
-              <Typography variant="body1">
-                Please enter your email address or Synapse user name and we'll
-                send you instructions to reset your password
-              </Typography>
+            <Box sx={{ position: 'relative' }}>
+              <IconButton
+                onClick={() => {
+                  history.goBack()
+                }}
+                sx={backButtonSx}
+              >
+                <ArrowBackIcon />
+              </IconButton>
+              <SourceAppLogo />
               <StyledFormControl
                 fullWidth
                 required
                 variant="standard"
                 margin="normal"
+                sx={formControlSx}
               >
+                <InputLabel shrink htmlFor="username" required>
+                  Email address or username
+                </InputLabel>
                 <TextField
                   fullWidth
                   id="username"
                   name="username"
                   onChange={e => setUserName(e.target.value)}
-                  placeholder="Email address-or-username"
                   value={userName || ''}
                 />
               </StyledFormControl>
               <Button
                 variant="contained"
                 fullWidth
                 onClick={handleResetPassword}
+                sx={buttonSx}
+                type="button"
+                disabled={!userName}
```

Comment:
Like the minimalistic view on the validation check (client-side regex check not as reliable/accurate as depending on server error response)